### PR TITLE
Separate testnet

### PR DESCRIPTION
**Pull request by BeaudanBrown**
Use different seed nodes for development vs production and strip 2 characters from pubkey on testnet

### CODE REVIEW

File: js/modules/loki_message_api.js [hunk 1]
```diff
@@ -112,7 +112,7 @@ class LokiMessageAPI {
     }
 
     const params = {
-      pubKey,
+      pubKey: window.getEnvironment() === 'production' ? pubKey : pubKey.substring(0, pubKey.length - 2),
```
Comment (sachaaaaa) [inline js/modules/loki_message_api.js]: Can you remind me what is the difference between production and development pubkeys?

Comment (BeaudanBrown) [reply js/modules/loki_message_api.js]: The plan for now is to just rip the last 2 characters off the end of the pubkey that is sent to the storage server
This won't affect anything besides the retrieve/store and pubkey -> swarm mapping (which I'm not fussed about on testnet)
I.e. the actual pubkey used for encryption/decryption will still be untouched inside the envelope
The reason for this is currently the only "validity" check being done on the pubkeys storage server side is that they are length 66, so to ensure that testnet pubkeys get rejected by mainnet nodes and vice versa we wanted to change the length of the keys

**Comment by BeaudanBrown** [comment]
Updated

File: js/modules/loki_rpc.js [hunk 2]
```diff
@@ -113,6 +113,13 @@ const rpc = (
   const portString = port ? `:${port}` : '';
   const url = `${address}${portString}${endpoint}`;
   // TODO: The jsonrpc and body field will be ignored on storage server
+  if (params.pubKey) {
+    // eslint-disable-next-line no-param-reassign
+    params = {
+      ...params,
+      pubKey: getStoragePubKey(params.pubKey),
+    };
+  }
```
Comment (sachaaaaa) [inline js/modules/loki_rpc.js]: It might be less efficient, but how about simply `params.pubKey = getStoragePubKey(params.pubKey)`?

File: js/modules/loki_message_api.js [hunk 3]
```diff
@@ -214,8 +214,9 @@ class LokiMessageAPI {
     let successiveFailures = 0;
     while (successiveFailures < MAX_ACCEPTABLE_FAILURES) {
       await sleepFor(successiveFailures * 500);
+      let result;
       try {
-        const result = await rpc(`https://${address}`, port, 'store', params);
+        result = await rpc(`https://${address}`, port, 'store', params);
```
Comment (sachaaaaa) [inline js/modules/loki_message_api.js]: Revert unnecessary changes above?

File: js/modules/loki_message_api.js (hunk 1 above)
Comment (sachaaaaa) [reply js/modules/loki_message_api.js]: Ok, thanks for the update !

File: js/modules/loki_rpc.js (hunk 2 above)
Comment (BeaudanBrown) [reply js/modules/loki_rpc.js]: Have to keep the if check to avoid undefined error but not sure why I was doing the `...` thing

File: js/modules/loki_message_api.js (hunk 3 above)
Comment (BeaudanBrown) [reply js/modules/loki_message_api.js]: Updated

File: js/modules/loki_rpc.js (hunk 2 above)
Comment (BeaudanBrown) [reply js/modules/loki_rpc.js]: Actually, reverting this because we need to make sure we aren't modifying the params outside the function call
